packages/components/atoms/f-button/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v3.0.1
```

Comment:
Can we make a ticket to add stories for the icon buttons to our visual regression (if we don't already)?

Reply:
Yes, I will write one now